pkgs/development/libraries/ucx/default.nix

```diff
@@ -33,10 +34,10 @@ stdenv.mkDerivation rec {
     autoreconfHook
     doxygen
     pkg-config
+    autoAddDriverRunpath
```

Comment:
Do we maybe keep it in optionals? Otherwise we might as well make stdenv always add the impure drivers runpath without any hooks